lib/importer/strategy-loader.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+
+let CustomImporter = class {
+  constructor() {}
+
+  execute(args) {}
+}
+
+const path = require('path'),
+        fs = require('fs-extra');
+
+let StrategyLoader = class {
+
+  constructor() {
+    this.basePath = './strategies';
+    this.strategies = [];
+    this.init();
+  }
+
+  init() {
+    let filePaths = fs.walkSync(this.basePath);
+    for (let filePath of filePaths) {
+      let baseName = path.basename(filePath, '.js'),
+          StrategyType = require(`${this.basePath}/${baseName}`);
+      this.strategies.push(new StrategyType());
+    }
+  }
+
+  load() {
+    for (let strategy of this.strategies) {
+      let chosen = strategy.determine();
```

Comment:
I'm not sure if I like the method name `determine`. What about `canExecute()` or `applies`?